Server/Modules/NoiseGenerator/Services/Perlin.cs

```diff
@@ -0,0 +1,396 @@
+﻿namespace Fracture.Server.Modules.NoiseGenerator.Services;
```

Comment:
to be fair i think we should sooner or later replace our hand-rolled implementation of the perlin noise generator with a NuGet package